build/cocos2d_libs.xcodeproj/project.pbxproj

```diff
@@ -17490,21 +17490,21 @@
 				HEADER_SEARCH_PATHS = "";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
 				LIBRARY_SEARCH_PATHS = (
-					"\"$(SRCROOT)/../external/png/prebuilt/ios\"",
```

Comment:
@diegorozen we don't have resource to support tvos. @pyrosphere will you maintain it?